e2e/playwright/point-click.spec.ts

```diff
@@ -451,3 +451,103 @@ sketch002 = startSketchOn(extrude001, seg03)
     }
   )
 })
+
+test(`Verify axis and origin snapping`, async ({
+  app,
+  editor,
+  toolbar,
+  scene,
+}) => {
+  // Constants and locators
+  // These are mappings from screenspace to KCL coordinates,
```

Comment:
Hopefully this makes it easier to map to @lf94's upcoming mouse position helpers that derive the expected KCL point values, by making it based on screen size and having the KCL representation paired with it.